packages/@lwc/perf-benchmarks/package.json

```diff
@@ -23,10 +23,10 @@
         "@lwc/synthetic-shadow": "5.0.0"
     },
     "devDependencies": {
-        "@best/cli": "^9.0.1",
-        "@best/runner-headless": "^9.0.1",
-        "@best/runner-remote": "^9.0.1",
-        "@lwc/rollup-plugin-node-resolve-legacy": "npm:rollup-plugin-node-resolve@5.2.0",
+        "@best/cli": "^10.0.0-alpha.0",
+        "@best/runner-headless": "^10.0.0-alpha.0",
+        "@best/runner-remote": "^10.0.0-alpha.0",
+        "@rollup/plugin-node-resolve": "15.2.3",
```

Comment:
```suggestion
```

This is already defined in the top-level `package.json`. No need to re-define it here.

Reply:
@nolanlawson Done!